apps/vscode-extension/src/intellisense/design-token.ts

```diff
@@ -10,9 +10,45 @@ type DesignTokenCache = {
 
 const readFile = async (filePat: string) => (await vscode.workspace.fs.readFile(vscode.Uri.parse(filePat))).toString();
 
+const getFilesPatternsFromProjectConfiguration = async (currentFile: string): Promise<string[] | undefined> => {
+  const angularFile = vscode.workspace.findFiles('angular.json');
+  const nxFile = vscode.workspace.findFiles('nx.json');
+  const projectFiles = vscode.workspace.findFiles('**/project.json')
+    .then((files) => files.filter((file) => path.relative(path.dirname(file.fsPath), currentFile).includes('..')));
```

Comment:
I am not sure to understand this part, do we want to check all `project.json` except the one related to the current file ?
or is a `!` missing ?